src/Custom/Assistants/MessageCollectionOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace OpenAI.Assistants;
+
+/// <summary>
+/// Represents addition options available when requesting a collection of <see cref="ThreadMessage"/> instances.
+/// </summary>
+public class MessageCollectionOptions
+{
+    /// <summary>
+    /// Creates a new instance of <see cref="MessageCollectionOptions"/>.
+    /// </summary>
+    public MessageCollectionOptions() { }
+
+    /// <summary>
+    /// The <c>order</c> that results should appear in the list according to
+    /// their <c>created_at</c> timestamp.
+    /// </summary>
+    public ListOrder? Order { get; init; }
```

Comment:
Per design guidance on loosening up accessors, should make all of these `set` and not `init`?

Reply:
I'm happy to move to whatever convention we decide on for the library on this.  I don't know if this one would have the same effect on the ASP.NET initialization sequence that using `init` for client options does, but I also don't feel strongly one way or another.  Have we finalized the decision around the convention for non-client options types?

Reply:
I'll merge this as-is since other options properties haven't been updated yet -- just so we don't end up with two approaches in a single library version.  Should be easy to blanket-address in a single project-wide search-replace.

Reply:
Sounds good. FYI: I synced with @ShivangiReja earlier today and she will be making the fix to change all `init` to `set`.